pyuvsim/profiling.py

```diff
@@ -69,13 +103,12 @@ def set_profiler(func_list=default_profile_funcs, rank=0, outfile_name='time_pro
         atexit.register(ofile.close)
         atexit.register(prof.print_stats, stream=ofile)
         if dump_raw:
-            outfile_raw_name = outfile_name + ".lprof"
-            if isinstance(dump_raw, str):
-                outfile_raw_name = dump_raw
+            outfile_raw_name = outfile_prefix + ".lprof"
             atexit.register(prof.dump_stats, outfile_raw_name)
-
+        setattr(prof, 'rank', rank)     # Add "rank" as an attribute to the profiler.
+        setattr(prof, 'meta_file', outfile_prefix + '_meta.out')
         prof.enable_by_count()
 
 
-def get_profiler():
+def get_profiler():  # pragma: nocover
```

Comment:
Let's add a comment to the lines you are leaving out of coverage because of the interaction between line_profiler and the coverage calculator to the effect that they are in fact covered by tests and that the `# pragma: nocover` can be removed when that bug is fixed.

Reply:
I've left comments here and in uvsim to explain the lack of coverage for now, which also reference the relevant issue on the line_profiler github page.